rotkehlchen/accounting/constants.py

```diff
@@ -67,140 +71,234 @@
         label='deploy',
         icon='mdi-swap-horizontal',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.BRIDGE_IN: EventCategoryDetails(
         label='bridge',
         icon='mdi-arrow-expand-up',
         color='red',
+        direction=EventDirection.OUT,
     ),
     EventCategory.BRIDGE_OUT: EventCategoryDetails(
         label='bridge',
         icon='mdi-arrow-expand-down',
         color='green',
+        direction=EventDirection.IN,
     ),
     EventCategory.GOVERNANCE: EventCategoryDetails(
         label='governance',
         icon='mdi-bank',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.DONATE: EventCategoryDetails(
         label='donate',
         icon='mdi-hand-heart-outline',
+        direction=EventDirection.OUT,
     ),
     EventCategory.RECEIVE_DONATION: EventCategoryDetails(
         label='receive donation',
         icon='mdi-hand-heart-outline',
+        direction=EventDirection.IN,
     ),
     EventCategory.RENEW: EventCategoryDetails(
         label='renew',
         icon='mdi-calendar-refresh',
+        direction=EventDirection.OUT,
     ),
     EventCategory.PLACE_ORDER: EventCategoryDetails(
         label='place order',
         icon='mdi-briefcase-arrow-up-down',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.TRANSFER: EventCategoryDetails(
         label='transfer',
         icon='mdi-swap-horizontal',
+        direction=EventDirection.IN,
     ),
     EventCategory.STAKING_REWARD: EventCategoryDetails(
         label='staking reward',
         icon='mdi-treasure-chest',
+        direction=EventDirection.IN,
     ),
     EventCategory.CLAIM_REWARD: EventCategoryDetails(
         label='claim reward',
         icon='mdi-gift',
+        direction=EventDirection.IN,
     ),
     EventCategory.LIQUIDATE: EventCategoryDetails(
         label='liquidate',
         icon='mdi-water',
+        direction=EventDirection.OUT,
     ),
     EventCategory.INFORMATIONAL: EventCategoryDetails(
         label='informational',
         icon='mdi-information-outline',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.CANCEL_ORDER: EventCategoryDetails(
         label='cancel order',
         icon='mdi-close-circle-multiple-outline',
         color='red',
+        direction=EventDirection.IN,
     ),
     EventCategory.REFUND: EventCategoryDetails(
         label='refund',
         icon='mdi-cash-refund',
+        direction=EventDirection.IN,
     ),
     EventCategory.FEE: EventCategoryDetails(
         label='fee',
         icon='mdi-account-cash',
+        direction=EventDirection.OUT,
     ),
     EventCategory.MEV_REWARD: EventCategoryDetails(
         label='mev',
         icon='mdi-star-box',
+        direction=EventDirection.IN,
     ),
     EventCategory.CREATE_BLOCK: EventCategoryDetails(
         label='new block',
         icon='mdi-cube-outline',
+        direction=EventDirection.NEUTRAL,
```

Comment:
I believe this is only used by the frontend actually so it can stay as this but maybe add a comment that we don't use it directly on the code